scipy/stats/_hypotests.py

```diff
@@ -686,34 +686,35 @@ def barnard_exact(table, alternative="two-sided", pooled=True, n=32):
     ----------
     table : array_like of ints
         A 2x2 contingency table.  Elements should be non-negative integers.
-    alternative : {'two-sided', 'less', 'greater'}, optional
-        Defines the alternative hypothesis.
-        The following options are available (default is 'two-sided'):
 
-        * 'two-sided'
-        * 'less': one-sided
-        * 'greater': one-sided
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the null and alternative hypotheses. Default is 'two-sided'.
+        Please see explanations in the Notes section below.
 
     pooled : bool, optional
-        Whether to compute score statistic with pooled variance (Student's
-        t-test) or unpooled variance (Welch's t-test). Default is ``True`` :
-        Statistic test is computed using pooled variance.
+        Whether to compute score statistic with pooled variance (as in
```

Comment:
I didn't understand the connection between the test statistic here and the t-statistic, so I thought you just mean to give examples of where pooled vs unpooled variance is used. Let me know if I'm wrong here.

Reply:
No you are not, Thank you ! :-)